extension/chrome/elements/passphrase.ts

```diff
@@ -101,8 +107,7 @@ Catch.try(async () => {
       }
     }
     if (atLeastOneMatched) {
-      BrowserMsg.send.passphraseEntry('broadcast', { entered: true });
-      BrowserMsg.send.closeDialog(parentTabId);
+      closeDialog();
```

Comment:
Notice that entered: true was now replaced with entered: false (meaning pass phrase was not entered). The iframe that is waiting for the passphrase may get confused. This should be tested.

Reply:
Ohh, Thanks, I missed that